mqtt/mqtt-broker/config/default.json

```diff
@@ -1,25 +1,21 @@
 {
     "transports": [
-        { 
+        {
             "tcp": {
-                 "address": "0.0.0.0:1883"
-            } 
+                "address": "0.0.0.0:1883"
+            }
         }
     ],
-    "inflight_messages": {
-        "max_count": 10
-    },
     "retained_messages": {
         "max_count": 1000,
         "expiration": "60d"
     },
     "session": {
         "expiration": "60d",
-        "messages": {
-            "max_message_size": "256kb",
-            "max_count": 1000,
-            "max_total_space": "16kb",
-            "when_full": "drop_new"
-        }                
+        "max_message_size": "256kb",
```

Comment:
Except that I'd prefer an embedded structure vs group1_settingA / group2_settingA style (e.g. message_expiration), I assume it can work. I think embedded structure is also better for extension. But I am not against the flat structure.